src/Audio.js

```diff
@@ -25,14 +25,14 @@ function Audio() {
     );
 
     const previousSong = () => (
-        <svg className="previousSong" width="21" height="21" viewBox="0 0 21 21" xmlns="http://www.w3.org/2000/svg">
+        <svg className="previousSong" width="21" height="21" viewBox="0 0 21 21" xmlns="http://www.w3.org/2000/svg" onClick={() => wavesurfer && wavesurfer.setTime(0)}>
```

Comment:
on prev click reset the time to 0, we can discuss this functionality more if we still want to support it